viewer/test/viewer-test-pptr.js

```diff
@@ -173,7 +173,8 @@ describe('Lighthouse Viewer', () => {
     });
 
     it('should support swapping locales', async () => {
-      function queryLocaleState() {
+      async function queryLocaleState() {
+        await viewerPage.waitForSelector('.lh-locale-selector');
```

Comment:
I didn't figure out the exact cause, but I guess the next line was executed too quickly and `.lh-locale-selector` wasn't ready.

When I switched to headfull chrome this test started passing.